src/EPiServer.SocialAlloy.ExtensionData/Membership/MembershipModeration.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using EPiServer.Social.Groups.Core;
 
-namespace EPiServer.SocialAlloy.Web.Social.Models.Moderation
+namespace EPiServer.SocialAlloy.ExtensionData.Membership
```

Comment:
Ok. The reason this was moved into its own proj (and consequently into its own namespace) is because these POCOs represent logic outside the scope of the just socialalloy / site domain.  Because we are so dependent on namespacing (i.e. the InlcudeSubclasses flag in composite criteria)  we are recommending that social users toss their POCOs into its own proj and include that into the site sln that they are building. The hope is that with this best practice their extensiondata namespaces will be less susceptible to changing over time. Additionally, by separating the extensiondata it should be very easy for a customer to reference the same POCOs in their staging site that they are referencing in their production site.  I don't think we call this out in the docs but we can in one of the tutorials.